lib/AST/PrettyStackTrace.cpp

```diff
@@ -263,3 +263,18 @@ void PrettyStackTraceGenericSignature::print(llvm::raw_ostream &out) const {
 void PrettyStackTraceSelector::print(llvm::raw_ostream &out) const {
   out << "While " << Action << " '" << Selector << "'";
 }
+
+void PrettyStackTraceDifferentiabilityWitness::print(
+    llvm::raw_ostream &out) const {
+  out << "While " << Action << ' ';
+  printDifferentiabilityWitnessDescription(out, Key, Context);
+}
+
+void swift::printDifferentiabilityWitnessDescription(
+    llvm::raw_ostream &out, const SILDifferentiabilityWitnessKey key,
+    ASTContext &Context, bool addNewline) {
```

Comment:
```suggestion
    bool addNewline) {
```
`ASTContext &Context` is unused, please remove the argument here and the corresponding field from `PrettyStackTraceDifferentiabilityWitness`.

Reply:
done